src/Container/AutowireFactory.php

```diff
@@ -36,7 +37,7 @@ private function getInjector(ContainerInterface $container)
     /**
      * Check creatability of the requested name
      *
-     * @param string $requestedName
+     * @param string|Stringable $requestedName
```

Comment:
Oooof, no, we should not really accept `Stringable` :-\

Reply:
That was just to keep BC with the config arrays. But I get your point.

Reply:
Nope you're right, this is actually extending the method signature which was not my intention.
Could we place a native `string` typehint? 

Some may argue that this would somehow be a BC break for user's who ignore the DocBlock and rely on the string cast behavior of this method.

Adding a native `string` type hint is valid imo, since everything else already violates the existing contract from the DocBlock.